raw-spans-grouper/helm/values.yaml

```diff
@@ -115,7 +116,7 @@ rawSpansGrouperConfig:
         producerMaxRequestSize: 10485760
         defaultProductionExceptionHandler: "org.hypertrace.core.kafkastreams.framework.exceptionhandlers.IgnoreProductionExceptionHandler"
         ignoreProductionExceptionClasses: "org.apache.kafka.common.errors.RecordTooLargeException"
-        numStandbyReplicas: 1
+        numStandbyReplicas: 0
```

Comment:
same comment here ^

Reply:
Setting to 1 as default value in main helm chart is inefficient. Fix it now.
Standby replicas are costly and needed only for applications which are high latency sensitive. They will help incase task failovers. Required only on need basis.

For more info
https://kafka.apache.org/10/documentation/streams/developer-guide/config-streams.html